lib/lndclient/LndClient.ts

```diff
@@ -51,6 +51,9 @@ class LndClient extends SwapClient {
   private channelBackupSubscription?: ClientReadableStream<lndrpc.ChanBackupSnapshot>;
   private invoiceSubscriptions = new Map<string, ClientReadableStream<lndrpc.Invoice>>();
   private maximumOutboundAmount = 0;
+  private maximumChannelOutboundAmount = 0;
```

Comment:
I think it would make sense to call thse `maxChannelOutboundAmount` and `maxChannelInboundAmount` and then rename the existing `maximumOutboundAmount` to `channelBalance` or `totalBalance`, which is a more accurate description I think.